pkg/kiosk/grafana_com_login.go

```diff
@@ -50,8 +51,17 @@ func GrafanaKioskGCOM(cfg *Config, messages chan string) {
 
 	// chromedp.WaitVisible(`//*[@href="login/grafana_com"]/i`, chromedp.BySearch),
 
+	headers := make(map[string]interface{})
```

Comment:
same for this file, this is for gcom (grafana cloud) login, and doesn't use basic auth, maybe this was not intended?

Reply:
see the following comment https://github.com/grafana/grafana-kiosk/pull/144#discussion_r1776464284
but if u think it's not needed then I'll just remove it.